packages/blockchain-info-components/src/Form/SelectInput/template.js

```diff
@@ -125,6 +125,11 @@ export const sharedSelect = css`
 
 const StyledSelect = styled(Select)`
   ${sharedSelect}
+
+  .dummy-input-wrapper {
+    position: absolute;
+    top: -1000px;
+  }
```

Comment:
So react-select always adds a `DummyInput` component to the `ValueContainer` but we weren't rendering it so the events that should be passed down/binded to the container where never there. This is for hiding that `DummyInput`.

Reply:
Did you try just using `visibility: hidden;` ?

Reply:
https://github.com/JedWatson/react-select/issues/2597